test/functional/test_framework/wallet_cli_controller.py

```diff
@@ -240,6 +240,15 @@ async def create_stake_pool(self,
     async def decommission_stake_pool(self, pool_id: str) -> str:
         return await self._write_command(f"staking-decommission-pool {pool_id}\n")
 
+    async def decommission_stake_pool_request(self, pool_id: str) -> str:
+        return await self._write_command(f"staking-decommission-pool-request {pool_id}\n")
+
+    async def sign_raw_transaction(self, transaction: str) -> str:
+        return await self._write_command(f"node-sign-raw-transaction {transaction}\n")
```

Comment:
Signing a transaction isn't a function for the node. We can put it under "account"?